docs/content/drafts/Label2.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Label
+componentId: label2
+status: Alpha
+source: https://github.com/primer/react/tree/main/src/Label2
+storybook: '/react/storybook?path=/story/label-label2--label'
+description: Use Label components to add contextual metadata to a design.
+---
+
+## Examples
+
+### Basic
+
+```javascript live noinline
+// import {Label} from '@primer/react/drafts'
+const {Label} = drafts // ignore docs silliness; import like that ↑
+
+render(<Label>Default</Label>)
+```
+
+### Appearances
+
+```javascript live noinline
+// import {Label} from '@primer/react/drafts'
+const {Label} = drafts // ignore docs silliness; import like that ↑
+render(
+  <>
+    <Label>Default</Label>
+    <Label scheme="primary">Primary</Label>
+    <Label scheme="secondary">Secondary</Label>
+    <Label scheme="accent">Info</Label>
+    <Label scheme="success">Success</Label>
+    <Label scheme="attention">Warning</Label>
+    <Label scheme="severe">Severe</Label>
+    <Label scheme="danger">Danger</Label>
+    <Label scheme="done">Done</Label>
+    <Label scheme="sponsors">Sponsors</Label>
+  </>
+)
+```
+
+### Sizes
+
+```javascript live noinline
+// import {Label} from '@primer/react/drafts'
+const {Label} = drafts // ignore docs silliness; import like that ↑
+render(
+  <>
+    <Label>Small (default)</Label>
+    <Label size="large">Large</Label>
+  </>
+)
+```
+
+## Props
+
+### Label
+
+<PropsTable>
+  <PropsTableRow
+    name="appearance"
```

Comment:
Looks like this prop is called `scheme` in the code.

Also, I'm curious if we should call this prop `variant` to be consistent with the Button API: https://primer.style/react/drafts/Button2#button

Reply:
Yup, this is just a mistake. Good catch!